seed/csharp-sdk/alias-extends/src/SeedAliasExtends/Core/IRequestOptions.cs

```diff
@@ -29,4 +30,15 @@ internal interface IRequestOptions
     /// The timeout for the request.
     /// </summary>
     public TimeSpan? Timeout { get; init; }
+
+    /// <summary>
+    /// Additional query parameters sent with the request.
+    /// </summary>
+    public IEnumerable<KeyValuePair<string, string>>? AdditionalQueryParameters { get; init; }
```

Comment:
I'd make this not nullable and add a default initializer with `Enumerable.Empty<KeyValuePair<string, string>>()